libraries/rush-lib/src/logic/pnpm/PnpmProjectManifestConfiguration.ts

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as path from 'path';
+import { JsonFile, JsonSchema, FileSystem } from '@rushstack/node-core-library';
+
+/**
+ * Pnpm config in package.json
+ * @beta
+ */
+export interface IPnpmProjectManifestConfigurationJson {
```

Comment:
```suggestion
interface IPnpmConfigJson {
```

Does this actually need to be exported?